doc/source/getting_started/comparison/includes/copies.rst

```diff
@@ -13,11 +13,3 @@ or overwrite the original one:
       df = df.sort_values("col1")
 
 .. note::
-
-   You will see an ``inplace=True`` keyword argument available for some methods:
```

Comment:
Should be fine to keep this one since we're discouraging inplace usage here?

Reply:
But with the wrong arguments, I'd rather add it again after finishing the pdep